### DISCUSSION TITLE
How to set a prefix to the pagination?

### DISCUSSION THREAD

**Discussion by freenandes** [marp-team/marp#485] [Q&A] [answered]
Right now it renders "X", the page number.

How do i customize the template to set "Page X of 20", or "X-20", or "X of 20 pages", and similar variations?

**Comment by yhatt** [comment, marked as answer]
👉 https://marpit.marp.app/theme-css?id=customize-content

```html
<style>
section::after {
  /* Add "Page" prefix, and total page number */
  content: 'Page ' attr(data-marpit-pagination) ' of ' attr(data-marpit-pagination-total);
}
</style>
```

Variants:

- `X-20`: `content: attr(data-marpit-pagination) '-' attr(data-marpit-pagination-total);`
- `X of 20 pages`: `content: attr(data-marpit-pagination) ' of ' attr(data-marpit-pagination-total) ' pages';`

**Reply by freenandes** [reply]
Thank you! I went so many times up and down the documentation on that part and not sure why i got blind for that specific part... It's very easy! Thank you!
